script/CreateMainnetProposals.s.sol

```diff
@@ -7,17 +7,17 @@ contract CreateMainnetProposal is WithChainIdValidation {
   constructor() WithChainIdValidation(1) {}
 }
 
-// Example proposal creation script for a single payload
+// // Example proposal creation script for a single payload
 // contract SinglePayloadProposal is CreateMainnetProposal {
 //   function run() external {
 //     GovHelpers.Payload[] memory payloads = new GovHelpers.Payload[](1);
 //     payloads[0] = GovHelpers.buildMainnet(
-//       0xd91d1331db4F436DaF47Ec9Dd86deCb8EEF946B4 // deployed cbETH payload
+//       address(0) // deployed cbETH payload
 //     );
 //     vm.startBroadcast();
 //     GovHelpers.createProposal(
 //       payloads,
-//       0x05097b8a0818a75c1db7d54dfd0299581cac0218a058017acb4726f7cc49657e
+//       0x05097b8a0818a75c1db7d54dfd0299581cac0218a058017acb4726f7cc49657e // TODO: Replace with actual hash
```

Comment:
Just curious if this was meant to change

Reply:
Probably best not to.